include/libdnf/comps/environment/sack.hpp

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright Contributors to the libdnf project.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2.1 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef LIBDNF_COMPS_ENVIRONMENT_SACK_HPP
+#define LIBDNF_COMPS_ENVIRONMENT_SACK_HPP
+
+#include "libdnf/common/sack/sack.hpp"
+#include "libdnf/common/weak_ptr.hpp"
+#include "libdnf/transaction/transaction_item_reason.hpp"
+
+#include <map>
+#include <string>
+
+
+namespace libdnf::comps {
+
+
+class Comps;
+class Environment;
+
+class EnvironmentSack;
+using EnvironmentSackWeakPtr = WeakPtr<EnvironmentSack, false>;
+
+
+class EnvironmentSack : public libdnf::sack::Sack<Environment> {
+public:
+    ~EnvironmentSack();
```

Comment:
It's still needed since there is the `WeakPtrGuard` and it's needed for `get_group_sack` in `Comps`. Well, at least until we decide what to do about the sacks.